coalib/bearlib/abstractions/Lint.py

```diff
@@ -1,339 +0,0 @@
-import os
-import re
-import shutil
-from subprocess import check_call, CalledProcessError, DEVNULL
-import tempfile
-
-from coalib.bears.Bear import Bear
-from coalib.misc.Decorators import enforce_signature
-from coalib.misc.Shell import escape_path_argument, run_shell_command
-from coalib.results.Diff import Diff
-from coalib.results.Result import Result
-from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
-
-
-class Lint(Bear):
```

Comment:
have already thought about it, will do :+1:
I want to delete it so noone gets encouraged to use the old class, as it's in fact buggy^^
But if you want I can only introduce the v2 Linter API, then we merge the adaptions on coala-bears, and then we delete the `Lint` class^^